src/core_plugins/kibana/public/management/react/index-pattern/index-pattern-create/index-pattern-create.component.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  KuiPageContent,
+  KuiPageContentBody,
+  KuiPageContentHeader,
+  KuiPageContentHeaderSection,
+  KuiTitle,
+  KuiText,
+  KuiForm,
+  KuiFormRow,
+  KuiSwitch,
+  KuiButton,
+  KuiFlexGroup,
+  KuiFlexItem,
+} from 'ui_framework/components';
+
+import { IndexPatternSearch } from './components/index-pattern-search';
+import { IndexPatternTimeFields } from './components/index-pattern-time-fields';
+import { IndexPatternResults } from './components/index-pattern-results';
+
+const IndexPatternCreate = ({
+  hasExactMatches,
+  isCreating,
```

Comment:
Just out of curiosity, what was the reason for keeping the state around fetched/fetching indices in the store instead of as local state?